content/influxdb/v2.1/notebooks/clean-data.md

```diff
@@ -10,31 +10,90 @@ menu:
     parent: Notebooks
 ---
 
-Learn how to create a notebook that normalizes data. Walk through the following example to create a notebook that does the following:
+Learn how to create a notebook that normalizes data.
+Data normalization is the process of modifying or cleaning data to be easier to
+work with. For example, adjusting numeric values to a uniform scale,
+modifying strings, etc.
 
-- Inputs sample data from NOAA
-- Normalizes sample data with a Flux script
-- Outputs normalized data to a bucket
+Walk through the following example to create a notebook that does the following:
 
-## Requirements
-- This example uses [NOAA water database data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data). Note that using this data counts towards your total usage.  
-- Create a destination bucket to write normalized data to. For details, see [Create a bucket](/influxdb/v2.1/organizations/buckets/create-bucket/).
+1.  Queries [NOAA NDBC sample data](/influxdb/v2.0/reference/sample-data/#noaa-ndbc-data)
+2.  Uses a Flux script notebook cell to normalizes degree-based wind directions
+    to cardinal directions.
+3.  Writes normalized data to a bucket
+
+{{< cloud-only >}}  
+{{% cloud %}}  
+**Note**: Using sample data counts towards your total InfluxDB Cloud usage.
+{{% /cloud %}}  
+{{< /cloud-only >}}
 
 ## Normalize data with a notebook
-1. Create a new notebook (see [Create a notebook](/influxdb/v2.1/notebooks/create-notebook/)).
-2. Add a **Metric Selector** cell to select a bucket to query data from:
-  - In the **Choose a bucket** dropdown, select your NOAA bucket.
-  - Select the **h2o_temperature** field.
-  - Select the **mean** function from the aggregate selector in the upper-right.
-3. Add a **Flux Transformation** cell with the following script to bring in data from the previous cell and normalize it:
-  ```sh
-  __PREVIOUS_RESULT__
-    |> map(fn: (r) => ({ r with _value: float(v: r._value) / 10.0 }))
-  ```
-4. Add an **Output to Bucket** cell:
-  - Select the destination bucket for your normalized data.
-5. Add a **Markdown** cell to add a note to your team about what this notebook does. For example, the cell might say, "This notebook makes `h20_temperature` easier to view alongside `water_level`."
-5. Click **Preview** in the upper left to verify that your notebook runs and preview the output.
-6. Run your notebook:
-  - Click **Run** to run the notebook and write to the output bucket a single time.
-  - To write continuously, click **Export as Task** in the upper right corner of the **Output to Bucket** cell. For details about working with tasks, see [Manage tasks](/influxdb/v2.1/process-data/manage-tasks/).
+1.  [Create a new notebook](/influxdb/v2.1/notebooks/create-notebook/).
+2.  In the **Build a Query** cell:
+
+    1.  In the **FROM** column under **{{% caps %}}Sample{{% /caps %%}},
+        select **NOAA National Buoy Data**.
```

Comment:
Do folks need to do something before they can see SAMPLE/ select NOAA...data? (I don't see in Cloud)

Reply:
Nope, this now exists natively in the FROM column (I thought that was pretty cool, BUT, it has to download the data every time, so it can be a little slow.